ballsdex/packages/players/cog.py

```diff
@@ -534,11 +534,18 @@ async def info(self, interaction: discord.Interaction):
         special = [x for x in ball if x.special is not None]
         trades = await Trade.filter(
             Q(player1__discord_id=interaction.user.id) | Q(player2__discord_id=interaction.user.id)
-        ).count()
+        ).values_list("player1__discord_id", "player2__discord_id")
+
+        trade_partners = set()
+        for p1, p2 in trades:
+            if p1 != interaction.user.id:
+                trade_partners.add(p1)
+            if p2 != interaction.user.id:
+                trade_partners.add(p2)
```

Comment:
```suggestion
            trade_partners.add(p1)
            trade_partners.add(p2)
        trade_partners.remove(interaction.user.id)
```


Just a suggestion instead of having to run the if check always